app/views/items/show.html.erb

```diff
@@ -1,8 +1,63 @@
+<% content_for :head_addition do %>
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.9.3/dist/leaflet.css"
+    integrity="sha256-kLaT2GOSpHechhsozzB+flnD+zUyjE2LlfWPgU04xyI="
+    crossorigin=""/>
+  <script src="https://unpkg.com/leaflet@1.9.3/dist/leaflet.js"
+    integrity="sha256-WBkoXOwTeyKclOHuWtc+i2uENFpDZ9YPdf5Hf+D7ewM="
+    crossorigin=""></script>
+<% end %>
+
 <p style="color: green"><%= notice %></p>
 
 <%= stylesheet_link_tag "item-show.css" %>
 <div id="<%= dom_id @item %>">
+<script>
+function getLocation() {
+  if (navigator.geolocation) {
+    navigator.geolocation.getCurrentPosition(saveUserPositionInHiddenForm, (error) => {
+      console.log(error);
+    });   
+  } else {
+    x.innerHTML = "Geolocation is not supported by this browser.";
+  }
+}
+
+function saveUserPositionInHiddenForm(position) {
+  document.getElementById('lat').value = position.coords.latitude;
+  document.getElementById('lng').value = position.coords.longitude;
+}
+</script>
   <%= render "item_largescreen" %>
   <%= render "item_smallscreen" %>
 </div>
+  
+
+<script>
+  function renderMap(div_id) {
+    // default values if item has no coordinates
+    var itemCoordinates = [32.71199, -94.12118];
```

Comment:
I was just about to change this to "Lake HPI" coordinates when I understood that these coordinates point at "Uncertain" - I love it! :smile: